websocket/controller.hh

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "seastar/core/future.hh"
+#include "seastar/core/shared_ptr.hh"
+#include "seastar/core/sharded.hh"
+
+#include "seastar/websocket/server.hh"
+
+#include "protocol_server.hh"
+#include "data_dictionary/data_dictionary.hh"
```

Comment:
Yikes! Not that I'm nitpicking for the minimally clean inclusion list, but this one caught my eye -- is it really required here?